lib/features/auth/data/api/model/response/AuthResponse.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:json_annotation/json_annotation.dart';
+
+import '../../../../domain/model/AppUser.dart';
```

Comment:
using full path more safe than relative path 
for some reasons like 
1. Code Refactoring Stability:
2. Consistency Across Modules:
for example -> 
**import 'package:your_project_name/domain/model/AppUser.dart';** 
is more safe than 
**import '../../../../domain/model/AppUser.dart';**